tests/Container/Exceptions/CannotAutowireExceptionTest.php

```diff
@@ -18,10 +19,11 @@ public function test_autowire_without_exception()
             $container = new GenericContainer();
 
             $container->get(ContainerObjectRequiringC::class);
-        } catch (CannotAutowireException $exception) {
-            $this->assertStringContainsString('string $prop in ContainerObjectC::__construct()', $exception->getMessage());
-            $this->assertStringContainsString('ContainerObjectC $c in ContainerObjectRequiringC::__construct()', $exception->getMessage());
-            $this->assertStringContainsString('Tests\Tempest\Container\Fixtures\ContainerObjectRequiringC', $exception->getMessage());
+            // TODO: Update the exception type.
+        } catch (Throwable $exception) {
+            $this->assertStringContainsString('[unresolved parameter: string $prop]', $exception->getMessage());
+            $this->assertStringContainsString('Tests\Tempest\Container\Fixtures\ContainerObjectC::__construct()', $exception->getMessage());
```

Comment:
Oh maybe you removed the reference to the method because it's listed as the second line?

Reply:
Yeah, basically it pulls the method name and combines it with the class name to show what failed. Then the attempted types show below that in a tree. Subject to change a little bit because I am still playing around with this tonight.